spec/lib/jobs/ansible-job-spec.js

```diff
@@ -3,8 +3,7 @@
 
 'use strict';
 
-var uuid = require('node-uuid'),
-    events = require('events');
+var uuid = require('node-uuid');
 
```

Comment:
I think you just wanted to see if I could rebase a single commit back out into multiple commit. Never one to stand down from a dare... Viola!